packages/orbit-components/src/Dialog/index.tsx

```diff
@@ -127,9 +127,7 @@ const Dialog = ({
   useLockScrolling(wrapperRef, lockScrolling);
   const ref = React.useRef<HTMLDivElement | null>(null);
   const theme = useTheme();
-  const transitionLength = React.useMemo(() => parseFloat(theme.orbit.durationFast) * 1000, [
-    theme.orbit.durationFast,
-  ]);
+  const transitionLength = parseFloat(theme.orbit.durationFast) * 1000;
```

Comment:
This const is actually only used inside another hook call (useEffect) so what we should do is to define it inside the hook and define `theme.orbit.durationFast` as the dependency for that hook. This way we avoid the `parseFloat` call on every render.

Something like:
```
  React.useEffect(() => {
    const transitionLength = parseFloat(theme.orbit.durationFast) * 1000;

    const timer = setTimeout(() => {
      setShown(true);
      if (ref.current) {
        ref.current.focus();
      }
    }, transitionLength);
    return () => clearTimeout(timer);
  }, [theme.orbit.durationFast]);
```

On line 135, and remove this line 130